owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/FileListAdapter.kt

```diff
@@ -203,9 +212,27 @@ class FileListAdapter(
                         it.Filename.text = file.fileName
                         it.fileListSize.text = DisplayUtils.bytesToHumanReadable(file.length, context)
                         it.fileListLastMod.text = DisplayUtils.getRelativeTimestamp(context, file.modificationTimestamp)
-                        it.fileListPath.apply {
-                            text = file.remotePath
-                            isVisible = !fileListOption.isAllFiles()
+                        if (fileListOption.isAvailableOffline() || (fileListOption.isSharedByLink() && fileItem.space == null)) {
+                            it.spacePathLine.path.apply {
+                                text = file.getParentRemotePath()
+                                isVisible = true
+                            }
+                            if (fileListOption.isAvailableOffline()) {
```

Comment:
This `if` is redundant, right? 

if it's `availableOffline` this is true.
If it's `isSharedByLink`, space will be null, so the s`pace?.let` clause won't be executed anyway

🤔

Reply:
True 💯